recipes/web.rb

```diff
@@ -0,0 +1,27 @@
+package 'st2web'
+
+node.default['nginx']['default_site_enabled'] = false
+
+# Ensure directory for Nginx SSL certificates is available
+directory '/etc/ssl/st2' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+end
+
+openssl_x509 '/etc/ssl/st2/st2.crt' do
```

Comment:
I followed https://docs.stackstorm.com/install/deb.html#install-webui-and-setup-ssl-termination, which uses a self-signed certificate. For the purpose of testing using the `bundle` recipe I think that this will do, but if we want users to bring their own certificate I guess that we can add an attribute and a conditional for this resource.